_includes/dev-menu.html

```diff
@@ -8,8 +8,9 @@
 			<li><a href="{{docu}}/developers/prerequisites/OSGi.html">OSGi</a>
 				<ul>
```

Comment:
Yes, I think so. I am working on another page about OSGi Declarative Services and pages about another topics.
I guess it will be good, if the OSGi topics are grouped together and the other topics remain in the Prerequisites sublevel.